### PR TITLE
Improving em estimation

### PR DESCRIPTION
**Pull request by GWeindel**
Solves issues #61 #62 #63

### CODE REVIEW

File: src/hsmm_mvpy/models.py [hunk 1]
```diff
@@ -89,6 +90,12 @@ def __init__(self, data, eeg_data=None, sfreq=None, offset=0, cpus=1, event_widt
         else: self.template = template
         self.events = self.cross_correlation(data.data.T)#adds event morphology
         self.max_d = self.durations.max()
+        self.em_method = em_method
+        if self.em_method == "mean":
```
Comment (jelmerborst) [inline src/hsmm_mvpy/models.py:94]: shouldn't there be an elif for "max" as well?

**Review by jelmerborst — APPROVED** [review]
great updates

File: src/hsmm_mvpy/models.py (hunk 1 above)
Comment (GWeindel) [reply src/hsmm_mvpy/models.py:94]: No because the max method doesn't need this new data representation

Comment (GWeindel) [reply src/hsmm_mvpy/models.py:94]: As a result, using max method is also lighter in terms of memory

**Pull request merged**